### ISSUE TITLE
"target" (SC 2.5.3 Target Size) - Proposal for change of wording

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wcag21#671]
https://www.w3.org/TR/WCAG21/#dfn-target

-- Current wording --

**target**

region of the display that will accept a touch action

If a portion of a touch target that does not perform the same action or go to the same page is overlapped by another touch target such that it cannot receive touch actions, then that portion is not considered a touch target for purposes of touch target measurements.

-- Suggested revisions --

**target**

region of the display that will accept a touch action

If two or more touch targets are overlapping in the same area, the area should not be included in the measurement of the target size, except when the overlapping targets perform the same action or open the same page.

--

This is hard to follow and translate. I wanted to remove redundancy and make this sentence more concise.

**Comment by awkawk** [comment]
(Official WG response)
Thank you for your suggestion. We agree that the definition of target was hard to parse and needed updating. We have changed "touch action" to "pointer action" to cover all pointer input, not just touch. We have also simplified the text explaining that parts of a target covered by another target must be excluded in target size measurements.
